### ISSUE TITLE
AuthAD plugin with PHP 8.1, still some errors on login screen

### ISSUE THREAD

**Issue by tqre-mc** [splitbrain/dokuwiki#3778]
I'm still getting some errors on loginscreen, using the latest patches for this plugin under Ubuntu 22.04 and PHP8.1, dokuwiki version
```
Warning: Undefined array key "REMOTE_USER" in /dokuwiki/lib/plugins/authad/auth.php on line 135
Warning: Undefined array key "REMOTE_USER" in /dokuwiki/lib/plugins/authad/auth.php on line 135
Warning: Undefined array key "REMOTE_USER" in /dokuwiki/lib/plugins/authad/auth.php on line 135
Warning: Undefined array key "REMOTE_USER" in /dokuwiki/lib/plugins/authad/auth.php on line 135
Warning: Undefined array key "REMOTE_USER" in /dokuwiki/lib/plugins/authad/auth.php on line 135
```

**Comment by Klap-in** [comment]
It is on purpose that REMOTE_USER is not set, if there is not a logged-in user.
Solution is indeed to replace `$_SERVER['REMOTE_USER']` with `$INPUT->server->str('REMOTE_USER')`.

Pull request is welcome.

**Comment by fiwswe** [comment]
I am fighting GitHub a bit here. My initial attempt at solving this was added to the unrelated PR #3774 which has not been merged yet. I think I need to uses branches on my copy of the DW repository to separate this but this is the first time I ran into this situation and I'll have to experiment and learn a bit.

Additionally there a small style error (a line was too long).

So I reverted the commit https://github.com/fiwswe/dokuwiki/commit/ce0feb5f5c38d0e698bd162105fbe61ceb27fca8 for now.
